src/_components/analytics/Analytics.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect } from 'react';
+import { EventContext } from '@snowplow/react-native-tracker';
+
+import { useTracking } from '../../_context';
+import { TRewardContext } from '../../_models/tracking';
+
+type TContexts = {
+  reward: TRewardContext;
+};
+
+export const schemes: Record<keyof TContexts, string> = {
+  reward: 'iglu:be.uitpas-mobile/reward/jsonschema/1-0-0',
+} as const;
+
+type TProps = {
+  contexts?: TContexts;
+  screenName?: string;
+};
+
+const Analytics = ({ screenName, contexts }: TProps) => {
+  const { trackScreenViewEvent } = useTracking();
+
+  useEffect(() => {
```

Comment:
wanneer moet er juist getracked worden? is dat on mount of on focus?

Reply:
onFocus, zou ik denken 🤔